### DISCUSSION TITLE
How to add z-index for headings select?

### DISCUSSION THREAD

**Discussion by pawelmidur** [mdx-editor/editor#311] [Q&A]
How to add z-index for headings select?
If I have editor inside MUI dialog, I don't see dropdown select for block type. How can I add `z-index` for this?

![image](https://github.com/mdx-editor/editor/assets/19294039/f44b9849-6ad9-4c24-b7a0-a686dc2cac6c)

**Comment by petyosi** [comment]
https://github.com/mdx-editor/editor/issues/305#issuecomment-1890862321
